assets/agenda/tests/utils.spec.js

```diff
@@ -184,9 +184,9 @@ describe('utils', () => {
             expect(listItems.hasOwnProperty('15-10-2018')).toBe(true);
             expect(listItems.hasOwnProperty('16-10-2018')).toBe(true);
             expect(listItems['14-10-2018']['_id']).toBe('foo');
-            expect(listItems['14-10-2018']['plan']['guid']).toBe('plan1');
+            expect(listItems['14-10-2018']['plan']['guid']).toBe('plan2');
```

Comment:
Shouldn't the list items for `14th` and `15th` be for `Plan1`?

Plan & Coverage Dates:
```typescript
plan1 = {
    'coverage1': '2018-10-15T04:00:00+0000',
    'coverage2': '2018-10-14T04:00:00+0000',
}
plan2 = {
    'coverage3': '2018-10-16T04:00:00+0000'
}
```

Reply:
it shows all coverages all the time now and that keyBy will pick the latest one which is `plan2` always